aws_cloudtrail_rules/aws_resource_made_public.py

```diff
@@ -64,6 +59,8 @@ def rule(event):
     # SecretsManager
     if event['eventName'] == 'PutResourcePolicy':
         policy = parameters.get('resourcePolicy', '{}')
-        return policy_is_not_acceptable(json.loads(policy))
 
-    return False
+    if len(policy) == 0:
+        return False
+
+    return policy_is_not_acceptable(json.loads(policy))
```

Comment:
What happens with policy is `None`? I don't know if that is possible in these events, but I ask because I've seen a few other aws events where a field exists but the value is null. Could this handle both cases (?): 
`return policy and policy_is_not_acceptable(json.loads(policy))`

Also, what happens if policy is never set? Should it be initialized with some default value (`None` ?) higher than the `if` statements scope?

Reply:
> What happens with policy is None? 
Good catch, yeah `len(None)` will return a `TypeError`
Python `==` handles that much better , I'll change it to `if policy == ""`

> Also, what happens if policy is never set? 
Note the `'{}' (default variable)` in the `parameters.get('policy', {}).get('policy', '{}')`, the built-in `dict.get` will return the specified default value if the key is not in that dict

Reply:
> Note the '{}' (default variable) in the parameters.get('policy', {}).get('policy', '{}'), the built-in dict.get will return the specified default value if the key is not in that dict

Gotcha 👍🏻   What about if the `event['eventName']` doesn't match any of the `if` statements - in that case, `policy` wouldn't 'exist', right? Do the if statements cover all possible `eventName`s?

Reply:
AFAIK `eventName` should always be there, but we could just apply the same logic we do for `parameters` to err on the side of caution